src/popup/router/components/PendingTxs.vue

```diff
@@ -21,15 +21,7 @@ export default {
   computed: mapState({
     transactions: ({ transactions: { pending } }) => pending
       .filter(({ amount, hash }) => !Number.isNaN(+amount) && hash)
-      .map(({
-        microTime, hash, tipUrl, ...tx
-      }) => ({
-        microTime,
-        hash,
-        tipUrl,
-        tx,
-        pending: true,
-      })),
+      .map((tx) => ({ ...tx, pending: true })),
```

Comment:
The original structure shouldn't be broken
https://github.com/aeternity/superhero-wallet/blob/develop/src/store/getters.js#L106 this won't work properly for example

Reply:
Didn't know that the sructure was broken (it was transaction.tx.tx instead of transaction.tx) and that you are fixing it here.